docs/Learning-Environment-Create-New.md

```diff
@@ -230,23 +194,17 @@ public override void CollectObservations(VectorSensor sensor)
 }
 ```
 
+### Taking Actions and Assigning Rewards
+
 The final part of the Agent code is the `Agent.OnActionReceived()` method, which
-receives the actions from the Brain and assigns the reward.
-
-### Actions
-
-The decision of the Brain comes in the form of an action array passed to the
-`OnActionReceived()` function. The number of elements in this array is determined by
-the `Vector Action` `Space Type` and `Space Size` settings of the
-agent's Brain. The RollerAgent uses the continuous vector action space and needs
-two continuous control signals from the Brain. Thus, we will set the Brain
-`Space Size` to 2. The first element,`action[0]` determines the force
```

Comment:
I think we should keep some of the instructions that are removed here, namely setting the action space to continuous and the space size to 2.

Reply:
Never mind, guess it's covered below.